Makefile

```diff
@@ -79,10 +79,6 @@ mongo-stop:
 mongo-shell:
 	docker exec -it mongo mongo
 
-.PHONY: docker-std docker-slim
-
-docker-slim: build
-	docker build --no-cache -t internal/tyk-gateway:slim -f ci/Dockerfile.slim .
-
-docker-std: build
-	docker build --no-cache -t internal/tyk-gateway:std -f ci/Dockerfile.std .
```

Comment:
@titpetric , just leave this two entries for now, since we will take care of slim removal in another PR, please fix the .PHONY statement to add your new docker target

Reply:
Done